CHANGELOG.md

```diff
@@ -0,0 +1,65 @@
+# Change Log
```

Comment:
this doesn't appear to be a log of changes for this project, presumably this is a CHANGELOG.md file from some dependency ... and hence doesn't belong in our project's repo